src/pages/patientView/PatientViewPage.tsx

```diff
@@ -79,8 +79,19 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     {
                         patientViewPageStore.setSampleId(query.sampleId as string);
                     }
-                    patientViewPageStore.patientIdsInCohort = ('navCaseIds' in query ? (query.navCaseIds as string).split(",") : []);
 
+                    let patientIdsInCohort = ('navCaseIds' in query ? (query.navCaseIds as string).split(",") : []);
+                    
+                    let multiStudyCohorts: string[] = patientIdsInCohort
+                    .filter(entity=> entity.split(':').length == 2);
+
+                    if(multiStudyCohorts.length > 0) {
+                        patientViewPageStore.patientIdsInCohort = multiStudyCohorts;
+                    } else {
+                        patientViewPageStore.patientIdsInCohort = patientIdsInCohort.map(entityId=>{
```

Comment:
why not just map patientIdsInCohort and where we don't detect a compound, make it compound (add studyId)?  Seems simpler to me.